en_us/quests/portfolio_introduction.md

```diff
@@ -0,0 +1,66 @@
+{% include '/version.md' %}
+
+# Portfolio Introduction
+
+## Quest objectives
+
+- Introduce Puppet the company
+- Give an overview of the Puppet portfolio of products
+- Describe Puppet Bolt
+
+## Getting started
+
+Now that you've been introduced to the Quest Guide for the Learning VM, this
+lesson introduces Puppet the company and its portfolio of products.
+
+Puppet was founded in 2005 by Luke Kanies and has offered its first commercial
+product, Puppet Enterprise (PE), since 2011.
+
+Puppet is a global company with headquarters in Portland and worldwide offices
+located in London, Belfast, Sydney, Tokyo and elsewhere. The company has over
+1000 enterprise customers, including 75 of the Fortune 100. Puppet has a robust
+user community that has built many thousands of contributed modules comprising
+over 7.5M lines of code. Additionally, tens of thousands of organizations use
+open-source Puppet to manage their server infrastructure.
+
+Over time, the company has expanded its offerings to the following portfolio
+of enterprise products:
+
+- Puppet Discovery: Identify your IT resources in real time and act on them
```

Comment:
I would omit this. With the 1.9 release, Discovery will sunset and VR--or whatever the new name is--will take its place.

Reply:
👍